src/main/java/net/rptools/maptool/client/functions/StringFunctions.java

```diff
@@ -443,11 +443,7 @@ public String format(String string, VariableResolver resolver, List<Object> args
     }
     m.appendTail(sb);
 
-    if (args == null) {
-      return sb.toString();
-    }
-
-    Object[] argArray = args.toArray();
+    Object[] argArray = args != null ? args.toArray() : new Object[] {};
```

Comment:
This can be cleaned up by changing the call site of `format()` instead. The casework on `size` there is now redundant as we can always pass along the tail of the parameters. And then the casework here would not be needed either.